tests/test_feeds.py

```diff
@@ -0,0 +1,220 @@
+import pytest
+import xml.etree.ElementTree as ET
+
+from django.utils.http import urlquote
+from django.core.urlresolvers import reverse
+
+from . import factories
+
+
+pytestmark = pytest.mark.django_db
+ns = {'feed': 'http://www.w3.org/2005/Atom'}
```

Comment:
if we are going to use "constants" (i use quotes because they don't really exist in Python), we it would be good to follow the convention of using all caps. Also, a more detailed name would help.

Note that this doesn't not necessarily apply to the `pytestmark`, because that is just a "module level marker", not a constant.